src/hooks/useBalances/useBalances.ts

```diff
@@ -13,9 +15,21 @@ export const useBalances = () => {
     const { address, ethBalance } = useSelector(
         (state: RootState) => state.wallet
     );
+    const { data } = useCurrencies(true);
+
+    const nativeCurrencies = useMemo(() => {
```

Comment:
In this app, the native currency balance is stored in `ethBalance` in the state. So this logic is for storing `ethBalance` into the native currency balance. 
I just noticed that `.find` was better than `filter`. it can change `nativeCurrencies` to `nativeCurrency`.

Reply:
Exactly!
I think we can rename `ethBalance` with `nativeCurrencyBalance` or just `balance`.

Reply:
@as472  I tried to use `useCurrencies` to get the nativeCurrency but it doesn't work here.
Do you know the way to get the nativeCurrency here?

Reply:
I fixed it using decimals from the public client.

Reply:
I renamed `ethBalance` and the related functions with `balance`.